app/javascript/redux/greetings/greetings.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+
+export const getGreetings = createAsyncThunk('greeting/getGreetings', async () => {
+    const response = await fetch('api/v1/greetings');
```

Comment:
- [OPTIONAL] I am not sure if this url from your backend would work for you, because the the correct url should be this `http://127.0.0.1:3000/api/v1/greetings`. I was only able to generate the random greetings after changing the url to the one I pasted here. Kindly look into and fix.